src/libopensc/card-mcrd.c

```diff
@@ -347,15 +347,18 @@ static int mcrd_init(sc_card_t * card)
 			if(apdu.sw1 != 0x90 && apdu.sw2 != 0x00)
 			{
 				sc_format_apdu(card, &apdu, SC_APDU_CASE_3, 0xA4, 0x04, 0x00);
-	                        apdu.lc = sizeof(EstEID_v35_AID);
-        	                apdu.data = EstEID_v35_AID;
-                	        apdu.datalen = sizeof(EstEID_v35_AID);
-                        	apdu.resplen = 0;
-	                        apdu.le = 0;
+							apdu.lc = sizeof(EstEID_v35_AID);
+							apdu.data = EstEID_v35_AID;
+							apdu.datalen = sizeof(EstEID_v35_AID);
+							apdu.resplen = 0;
+							apdu.le = 0;
```

Comment:
please fix the indenting

Reply:
It is fixing indenting, before there was spaces and tabs mixed

Reply:
or you mean leave the mixed indenting as it is?

Reply:
There are too many tabs. Look at GitHub's diff, you'll see the problem.